include/oneapi/dpl/pstl/algorithm_fwd.h

```diff
@@ -778,6 +778,22 @@ void
 __pattern_sort_by_key(__parallel_tag<_IsVector>, _ExecutionPolicy&&, _RandomAccessIterator1, _RandomAccessIterator1,
                       _RandomAccessIterator2, _Compare);
 
+//------------------------------------------------------------------------
+// stable_sort_by_key
+//------------------------------------------------------------------------
+
+template <class _Tag, typename _ExecutionPolicy, typename _RandomAccessIterator1, typename _RandomAccessIterator2,
```

Comment:
For consistency we write `typename `(instead of class), where it is appicable.

Reply:
Fixed